lib/db.js

```diff
@@ -4,8 +4,10 @@ const _ = require('lodash');
 const async = require('async');
 const utils = require('./utils');
 const Redis = require('ioredis');
-const EventEmitter = require('events').EventEmitter;
+const TTLCache = require('@isaacs/ttlcache');
 const { validateParams } = require('./validation');
+const {parseInt} = require("lodash");
```

Comment:
Why loadash's `parseInt`  instead of vanila-javascript?

Reply:
Whoa. I think this was my editor 🤔 I don't use `"`. Will adjust.